userspace/libscap/scap_procs.c

```diff
@@ -1327,8 +1327,11 @@ void scap_proc_print_table(scap_t* handle)
 const char *scap_strerror(scap_t *handle, int errnum)
 {
 	int rc;
-
+#if defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32)
+	if((rc = strerror_s(handle->m_strerror_buf, SCAP_LASTERR_SIZE, errnum) != 0))
+#else
 	if((rc = strerror_r(errnum, handle->m_strerror_buf, SCAP_LASTERR_SIZE) != 0))
+#endif
```

Comment:
I'd probably make a wrapper to hide that, just to avoid duplicating the parameters, but that's just me :)

```
#ifdef SYSDIG_WIN
static inline int strerror_r(...)
{
    return strerror_s(...);
}
#endif
```
or
```
#ifdef SYSDIG_WIN
#define strerror_r(errnum, buf, size) strerror_s(buf, size, errnum)
#endif

Reply:
Good idea. I took the second option.